### DISCUSSION TITLE
Setting up for development - "internal" dependencies

### DISCUSSION THREAD

**Discussion by WillowP** [rjsf-team/react-jsonschema-form#3050] [Q&A]
I'm trying to set up for development, and have not worked on a multi-package project before.

If I run `npm i --include=dev` in the root directory, and then `npm test`, the tests fail with the following errors:

```
   Tasks not run because their dependencies failed or --nx-bail=true:
   
   - @rjsf/chakra-ui:test
   - @rjsf/core:test
   - @rjsf/fluent-ui:test
   - @rjsf/material-ui:test
   - @rjsf/mui:test
   - @rjsf/semantic-ui:test
   - @rjsf/utils:test
   - @rjsf/validator-ajv6:test

   Failed tasks:
   
   - @rjsf/antd:test
   - @rjsf/bootstrap-4:test
```

An example of an error from a single test:
```
Cannot find module '@rjsf/utils' or its corresponding type declarations.
```

I think there must be something I need to do to get the various `packages` to be able to access each other, but I am not sure what that is. Any help is appreciated :)

**Comment by heath-freenome** [comment]
It is likely that, because prior to the release of the beta, npm did not have a `@rjsf/utils` package in its catalog
